lib/commands/source.js

```diff
@@ -13,13 +13,15 @@ const commands = {
       return await this.executeAtom('execute_script', [script, []]);
     }
 
-    if ((await this.settings.getSettings()).useJSONSource) {
-      const srcTree = await this.mobileGetSource('json');
+    const settings = await this.settings.getSettings();
+    const excludedAttributes = settings.pageSourceExcludedAttributes;
```

Comment:
this might be simplified using objects deconstruction 

```
const {
  pageSourceExcludedAttributes: excludedAttributes
} = await this.settings.getSettings();
```

Reply:
done